lib/ansible/playbook/role_include.py

```diff
@@ -53,7 +53,7 @@ class IncludeRole(TaskInclude):
 
     # private as this is a 'module options' vs a task property
     _allow_duplicates = FieldAttribute(isa='bool', default=True, private=True)
-    _public = FieldAttribute(isa='bool', default=False, private=True)
+    _public = FieldAttribute(isa='bool', default=False)
```

Comment:
Is this more of a keyword than allow_duplicates? It looks like a module arg to users so hiding it from ansible-doc still makes sense to me.

Reply:
u right